app/controllers/spree/admin/taxjar_settings_controller.rb

```diff
@@ -1,10 +1,6 @@
 module Spree
   module Admin
     class TaxjarSettingsController < Spree::Admin::BaseController
-      def edit
```

Comment:
Why is this being removed? It's not mentioned in the commit message.

Reply:
Added a note to the commit message!